debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerErrorHandler.java

```diff
@@ -34,7 +34,7 @@ protected boolean isRetriable(Throwable throwable) {
                         || throwable.getMessage().contains("Connection reset")
                         || throwable.getMessage().contains("SHUTDOWN is in progress")
                         || throwable.getMessage().contains("The server failed to resume the transaction")
-                        || throwable.getMessage().contains("Connection refused (Connection refused)")
```

Comment:
Why is this one removed?

Reply:
This will be covered by the newly added pattern. This is what the full message looks like:
> The TCP/IP connection to the host localhost, port 1433 has failed. Error: "Connection refused (Connection refused). Verify the connection properties. Make sure that an instance of SQL Server is running on the host and accepting TCP/IP connections at the port. Make sure that TCP connections to the port are not blocked by a firewall.".

Reply:
Ah, that's great. Thanks!